README.md

```diff
@@ -0,0 +1,38 @@
+# unity-persistent-data-store
+
+## What is it?
+A library wrapping Unity's persistentDataPath access, to store arbitrary serializable objects by their type, or unique IDs.
+
+Features:
+* Persistent storage of serializable objects.
+
+## How to use it.
+Call the PersistentDataStore.SerializedData API.
+
+### Saving and loading singleton objects (only one per application)
+
+if (SerializedData.Exists<MainConfig>())
```

Comment:
Wrap this in code blocks for better readability

Reply:
Done